### PR TITLE
Feat: dev deployment pipeline

### LINKED ISSUE #525: Automatic deployment of Dev env on merge to develop

### PR DESCRIPTION
**Pull request by kmaus-near**
Closes #525

### CODE REVIEW

File: .github/workflows/multichain-dev.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+name: Deploy Multichain Dev.
+on:
+  pull_request:
+    types: [closed]
```
Comment (ppca) [inline .github/workflows/multichain-dev.yml:4]: should we differentiate between closed and merged?

**Review by ppca — APPROVED** [review]
one comment only about PR type

File: .github/workflows/multichain-dev.yml (hunk 1 above)
Comment (kmaus-near) [reply .github/workflows/multichain-dev.yml:4]: That's what line 14 and 34 do 😄

**Review by volovyks — COMMENTED** [review]
LGTM, but let's add contract deployment, it would be very inconvenient to do that manually all the time.

**Comment by ppca** [comment]
> LGTM, but let's add contract deployment, it would be very inconvenient to do that manually all the time.

yep. contract is coming up in another PR

**Pull request merged**